devel/lib/lib.sh

```diff
@@ -23,8 +23,8 @@ export REPO_ROOT="$LIB_ROOT/../.."
 
 export SKIP_BUILD_ADDON_IMAGES="${SKIP_BUILD_ADDON_IMAGES:-}"
 export KIND_CLUSTER_NAME="${KIND_CLUSTER_NAME:-kind}"
-# Default Kubernetes version to use to 1.24
-export K8S_VERSION=${K8S_VERSION:-1.24}
+# Default Kubernetes version to use to 1.25
```

Comment:
I don't think we should change the default version until the next version of cert-manager as it can introduce new issues  We don't want to spend any time on investigating if something was introduced in 1.25 when we are close to cutting a new release.

Reply:
> I would rather that we used 1.25.0 for the latest release because without it, we can't guarantee fully comparability with the latest Kubernetes version.
> 
> I think it is fine for us to push back the release a week to make sure CI remains green.

There is also the fact that local development and testing has been done primarily on <= 1.24.

If you are confident in switching the default version at this point in time then I think we can go ahead. You have more historical context.

Reply:
Regardless, I think the path forward is to add 1.25 jobs and the necessary configurations - make sure it all works and then switch the default version in a separate PR.

I do not think 1.8 or 1.9 need to be tested on 1.25, as it is not supported by those version.

Reply:
Yes I think the right thing to do is bump kube version in cert-manager to v1.25 for this upcoming release.

Agreed that we do not need 1.25 tests for 1.8 & 1.9